metabase/metabase
# Clean up `QuestionRowCount` component

Cleans up the `QuestionRowCount` component before we change it in #29454

* adds test coverage
* converts the component to TypeScript
* extracts styled-components
* refactors the component to avoid nesting
* makes it use an HTML `button` instead of a `span` when the row count is clickable

### How to verify

1. Open any question
2. Play around with the row count component and ensure both look and behavior match the state of `master`

### Demo

https://user-images.githubusercontent.com/17258145/227524145-360ac8de-fe77-4a4d-a275-ab96ff469bae.mp4

### Checklist
